backend/src/ee/routes/v1/sso.ts

```diff
@@ -30,6 +30,7 @@ router.get(
 router.get("/redirect/github", authLimiter, (req, res, next) => {
   passport.authenticate("github", {
     session: false,
+    scope: [ 'user:email' ],
```

Comment:
nit: unrelated change right?

Reply:
Yeah. It must’ve got caught in the update for this but unrelated.